src/main/java/com/dnd/namuiwiki/domain/survey/AnswerController.java

```diff
@@ -0,0 +1,33 @@
+package com.dnd.namuiwiki.domain.survey;
+
+import com.dnd.namuiwiki.common.dto.ResponseDto;
+import com.dnd.namuiwiki.domain.jwt.JwtAuthorization;
+import com.dnd.namuiwiki.domain.jwt.dto.TokenUserInfoDto;
+import com.dnd.namuiwiki.domain.survey.type.Period;
+import com.dnd.namuiwiki.domain.survey.type.Relation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/v1/answers")
+public class AnswerController {
+    private final SurveyService surveyService;
+
+    @GetMapping
+    public ResponseEntity<?> getAnswersByQuestion(
+            @JwtAuthorization TokenUserInfoDto tokenUserInfoDto,
+            @RequestParam(name = "questionId") String questionId,
+            @RequestParam(name = "period", required = false, defaultValue = "TOTAL") String period,
+            @RequestParam(name = "relation", required = false, defaultValue = "TOTAL") String relation,
```

Comment:
```suggestion
            @RequestParam(name = "period", required = false, defaultValue = "TOTAL") Period period,
            @RequestParam(name = "relation", required = false, defaultValue = "TOTAL") Relation relation,
```

stringToEnumConverter가 있어서 enum 타입으로 자동으로 변환됩니다!